core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestUtils.java

```diff
@@ -28,6 +30,11 @@
 public final class S3RestUtils {
   private static final Logger LOG = LoggerFactory.getLogger(S3RestUtils.class);
 
+  public static final String S3_STANDARD_STORAGE_CLASS = "STANDARD";
```

Comment:
either put all the s3 related constants into the existing `alluxio.Constants` or into an independent `S3Constants`?

Reply:
done, moved to `S3Constants`